zilencer/urls.py

```diff
@@ -42,5 +39,4 @@
 urlpatterns = [
     path("api/v1/", include(push_bouncer_patterns)),
     path("api/v1/", include(billing_patterns)),
-    path("remotes/billing/<signed_billing_access_token>", remote_server_billing_ui),
```

Comment:
This now turns into `"remote-billing-login/<signed_billing_access_token>", remote_server_billing_finalize_login`. Moved it to `corporate/urls.py`, since that's where our user-facing billing endpoints live. 

I'm keeping `remote_server_billing_entry` in `zilencer/urls.py`, because that's essentially a bouncer endpoints - users don't interact with it - remote servers do, and their authenticate with their bouncer credentials. Hopefully this convention makes sense.